### PR TITLE
Adding docs for offline plugin

### PR DESCRIPTION
**Pull request by langsmith**
Resolves https://github.com/mapbox/android-docs/issues/266 by adding written documentation, sidebar setup, and other boilerplate for the offline plugin that's recently been released on Maven.

screencapture-localhost-8080-android-docs-plugins-overview-offline-2018-04-18-15_49_24

[Offline plugin javadocs have already been added](https://github.com/mapbox/android-docs/pull/425) too.
screen shot 2018-04-18 at 3 55 37 pm

### CODE REVIEW

**Comment by cammace** [comment]
Can we also add to the API drop down menu:

- Localization
- Clustering

**Comment by langsmith** [comment]
@cammace, I'll add missing localization and clustering info/menu stuff in separate prs. Ticketed at https://github.com/mapbox/android-docs/issues/434 and https://github.com/mapbox/android-docs/issues/435

File: src/pages/plugins/overview/offline.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  - title: "Add the Offline Plugin"
+prependJs:
+  - |
+    import {
+      OFFLINE_PLUGIN_VERSION
+    } from '../../../constants';
+---
+# Offline
+
+The Mapbox Offline Plugin for Android automatically does the downloading and managment of map tiles for offline Android device use. Especially if you're building a project for outdoors use, a user's device won't always have a strong enough internet connection to download and view map tiles. Once given certain information, this plugin handles the rest of the work that's required for fetching and storing map tiles for any region of the world that you want.
```
[the remark this answers is not in the document]
Comment (langsmith) [reply src/pages/plugins/overview/offline.md]: Ok, I reworded the first paragraph a bit

**Comment by langsmith** [comment]
@colleenmcginnis , I've added a paragraph about the estimator. It's the same paragraph that's found in  https://www.mapbox.com/android-docs/map-sdk/overview/offline/#limitations

**Comment by langsmith** [comment]
Anything else you two?

**Pull request merged**